src/mesh/azext_mesh/tests/latest/recordings/test_service_replica_commands.yaml

```diff
@@ -0,0 +1,548 @@
+interactions:
```

Comment:
Add the instruction how to run tests for Mesh in README page as a reference for future contributions from others.